python/lsst/obs/base/butler_tests.py

```diff
@@ -127,8 +127,11 @@ def test_exposureId_bits(self):
     def _test_exposure(self, name):
         if self.dataIds[name] is unittest.SkipTest:
             self.skipTest('Skipping %s as requested' % (inspect.currentframe().f_code.co_name))
-
         exp = self.butler.get(name, self.dataIds[name])
+
+        exp_md = self.butler.get(name+"_md", self.dataIds[name])
+        self.assertEqual(type(exp_md), type(exp.getMetadata()))
```

Comment:
Can you test that the objects themselves are equal (as they should be)? (I expect there's no `PropertyList::operator==`, so this is annoying, but it really needs to be done.)

Reply:
Unfortunately, no.  As Jim explained in science pipelines some time ago, we are not completely strict about round tripping the metadata, so there are a few cards in one that aren't in the other.
His suggestion was that this is the best we can do at the moment (and is better than we were doing before).